modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/BlockStreamTest.kt

```diff
@@ -38,89 +40,109 @@ class BlockStreamTest : IrohaTest<Iroha2Client>() {
-        blocksResult.collect { block -> blocks.add(block) }
+        subscription.receiveBlocking<VersionedBlockMessage>(actionId).collect { block -> blocks.add(block) }
 
         val expectedSize = NewAccountWithMetadata().block.transactions.sumOf { it.size }
-        var instructions = checkBlockStructure(blocks[0], 1, GENESIS, GENESIS, expectedSize)
-        val registerDomain = instructions[0].cast<InstructionBox.Register>().extractDomain().id.name.string
+        var isi = checkBlockStructure(blocks[0], 1, GENESIS, GENESIS, expectedSize)
+        val registerDomain = isi[0].cast<InstructionBox.Register>().extractDomain().id.name.string
+
         assertEquals(DEFAULT_DOMAIN_ID.asString(), registerDomain)
+        assertEquals(ALICE_ACCOUNT_ID.asString(), isi[1].extractAccount().id.asString())
+        assertEquals(BOB_ACCOUNT_ID.asString(), isi[2].extractAccount().id.asString())
         assertEquals(
-            ALICE_ACCOUNT_ID.asString(),
-            instructions[1].cast<InstructionBox.Register>().extractAccount().id.asString(),
-        )
-        assertEquals(
-            BOB_ACCOUNT_ID.asString(),
-            instructions[2].cast<InstructionBox.Register>().extractAccount().id.asString(),
-        )
-        assertEquals(
-            "foo$ACCOUNT_ID_DELIMITER$DEFAULT_DOMAIN",
-            instructions[3].cast<InstructionBox.Register>().extractAccount().id.asString(),
+            "${NewAccountWithMetadata.ACCOUNT_NAME.string}$ACCOUNT_ID_DELIMITER$DEFAULT_DOMAIN",
+            isi[3].extractAccount().id.asString(),
         )
 
-        instructions = checkBlockStructure(
-            blocks[1],
-            2,
-            DEFAULT_DOMAIN,
-            BOB_ACCOUNT,
-            1,
-        )
-        var newAssetDefinition = instructions[0].cast<InstructionBox.Register>().extractAssetDefinition()
+        isi = checkBlockStructure(blocks[1], 2, DEFAULT_DOMAIN, BOB_ACCOUNT, 1)
+        var newAssetDefinition = isi[0].cast<InstructionBox.Register>().extractAssetDefinition()
         assertNotNull(newAssetDefinition)
         assertEquals(newAssetName, newAssetDefinition.id.name.string)
         assertEquals(DEFAULT_DOMAIN, newAssetDefinition.id.domainId.asString())
 
-        // get the last block second time
-        blocksResult = client.subscribeToBlockStream(2, 1)
-        blocks = mutableListOf<VersionedBlockMessage>()
-        blocksResult.collect { block -> blocks.add(block) }
-        instructions = checkBlockStructure(
-            blocks[0],
-            2,
-            DEFAULT_DOMAIN,
-            BOB_ACCOUNT,
-            1,
-        )
-        newAssetDefinition = instructions[0].cast<InstructionBox.Register>().extractAssetDefinition()
-        assertNotNull(newAssetDefinition)
-        assertEquals(newAssetName, newAssetDefinition.id.name.string)
-        assertEquals(DEFAULT_DOMAIN, newAssetDefinition.id.domainId.asString())
-    }
+//        blocks = mutableListOf()
```

Comment:
provide comment why it's commented or remove the commented part

Reply:
we have `@Issue` annotation on this test